packages/select/src/useSelect.ts

```diff
@@ -20,7 +20,7 @@ import {
 } from '@element-plus/utils/util'
 import isEqual from 'lodash/isEqual'
 
-const ELEMENT = { size: 'medium' }
+const ELEMENT = { size: '' }
```

Comment:
Reason?

Reply:
the select default size is '', height '40px', if default 'medium', height is '36px', so the default height is different in form

Reply:
This one is a general bug, needed to be fixed for all form items.

Reply:
Validation demo in doc, input near select, so i found this

Reply:
You can refer to #417 , ElementPlus now needs a global configuration to make all attributes required from a single source